helm/prometheus-operator-app/templates/prometheus-operator/crds.yaml

```
@@ -1,4 +1,4 @@
-{{- if and .Values.prometheusOperator.enabled .Values.prometheusOperator.createCustomResource -}}
+{{- if and .Values.prometheusOperator.enabled .Values.prometheusOperator.createCustomResource (eq .Release.Service "Tiller") -}}
```

```suggestion
{{- if and (eq .Release.Service "Tiller") .Values.prometheusOperator.enabled .Values.prometheusOperator.createCustomResource  -}}
```
I think we can bring up this front of the statement. WDYT?